app/views/nav/toggle_collapse_sidebar.js.erb

```diff
@@ -0,0 +1,9 @@
+$('.collapse-nav').find('input[type="submit"]').prop('disabled', false).removeClass("disabled")
```

Comment:
Even though it's a `js.erb` file? I though haml was just for HTML templates, but let me know if I'm misunderstanding that. I don't usually use HAML...